### PR TITLE
docs(recipe): Using global CSS files

### LINKED ISSUE #14892: [docs][recipes] Styling: Add Global CSS files
## Summary

To improve key learning workflows in the Recipes section of the docs, the "Styling" recipe should be broken up into smaller, more actionable parts. This issue is about the first item, **standard global CSS files** [without a layout component](https://www.gatsbyjs.org/docs/global-css/#adding-global-styles-without-a-layout-component).

The current recipe has this text (it will be replaced by Styling recipes inline): 

> There are so many ways to add styles to your website; Gatsby supports almost every possible option, through official and community plugins.
> - Walk through adding global styles to an example site in tutorial part two
>    - More on global styles with standard CSS files
>    - More on global styles with CSS-in-JS
>    - More on global styles with CSS files and no layout component
> - Use the CSS-in-JS library Glamor
> - Use the CSS-in-JS library Styled Components
> - Use CSS Modules

The actionable version of this **global CSS recipe** should show how to work with a vanilla CSS file _inline_ as concisely as possible. It should include code snippets showing a short CSS file and how it can be imported into `gatsby-browser.js` (like on https://www.gatsbyjs.org/docs/global-css/#adding-global-styles-without-a-layout-component).

Recipes now follow this format:

- Requirements or prerequisites 
- Step-by-step instructions
- Relevant links to any relevant information
    - Link to a live example
    - More on using standard global CSS files https://www.gatsbyjs.org/docs/global-css/

This recipe will go into the "Styling" section of [`docs/docs/recipes.md`](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/recipes.md), replacing the existing text. There is a template in the contributing docs: https://www.gatsbyjs.org/contributing/docs-templates/#recipes

For reference, the new recipe collection is also listed in a spreadsheet: https://docs.google.com/spreadsheets/d/1yNUQi_-3TsuDtE98hCIJQvLZ6X1YHlDmC-EJwOGxiHk/edit#gid=1252007048

### PR DESCRIPTION
**Pull request by mehtavishwa30**
## Description

This is a recipe for the first item `Using global CSS files without a Layout component` in the `Styling` section of the Recipes page. 

## Related Issues

Fixes #14892

### CODE REVIEW

File: docs/docs/recipes.md [hunk 1]
```diff
@@ -265,6 +265,46 @@ export default ({ pageContext: { dog } }) => (
...
+html {
+  background-color: lavenderblush;
+}
+
+p {
+  color: peachpuff;
+}
+```
+
+2. Import the global CSS file in the `gatsby-config.js` file such as the following:
```
Comment (sidharthachatterjee) [inline docs/docs/recipes.md]:
```suggestion
2. Import the global CSS file in the `gatsby-browser.js` file such as the following:
```

File: docs/docs/recipes.md [hunk 2]
```diff
@@ -265,6 +265,46 @@ export default ({ pageContext: { dog } }) => (
...
+
+```javascript:gatsby-config.js
```
Comment (sidharthachatterjee) [inline docs/docs/recipes.md]:
```suggestion
```javascript:gatsby-browser.js
```

**Comment by marcysutton** [comment]
Thanks for this recipe! However we need to make an update to it: the `peachpuff` foreground color on a `lavenderblush` background has a very low contrast ratio (1.19:1, when it should be 4.5:1). Something like `#895d37` for the text color would stand out a lot more.

File: docs/docs/recipes.md (hunk 1 above)
Comment (mehtavishwa30) [reply docs/docs/recipes.md]: Oh yes, sorry about that! Thanks for pointing this out. Applying the suggestions right away!

**Comment by mehtavishwa30** [comment]
Thanks for drawing my attention to this, @marcysutton! I should update the recipe with a better contrast ratio in the code snippet.

**Comment by mehtavishwa30** [comment]
Also, I was thinking if this item should come after the `Using global CSS files with a Layout component`, so that the structure is consistent with the content in the `Reference Guides` [here](https://www.gatsbyjs.org/docs/global-css/#adding-global-styles-without-a-layout-component)?

**Comment by marcysutton** [comment]
Thanks, @mehtavishwa30! I think the order is fine. I'd actually list this one before the layout component one, personally, since it has one less concept involved.

**Comment by mehtavishwa30** [comment]
Makes sense, @marcysutton! Thanks!

**Pull request merged**